Library/Homebrew/formula_installer.rb

```diff
@@ -942,14 +942,41 @@ def post_install
     @show_summary_heading = true
   end
 
-  def pour
+  def fetch_dependencies
+    deps = compute_dependencies
+
+    return if deps.empty? || ignore_deps?
+
+    deps.each do |dep|
+      fetch_dependency(dep.to_formula)
+    end
+  end
+
+  def fetch_dependency(dep)
```

Comment:
I'd be tempted to inline this method in `deps.each` above as it's not used elsewhere.